src/prefect/tasks/aws/parameter_store_manager.py

```diff
@@ -0,0 +1,58 @@
+from prefect.tasks.secrets.base import SecretBase
+from prefect.utilities.aws import get_boto_client
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class AWSParametersManager(SecretBase):
+    """
+    Task for retrieving values from AWS SSM Parameters Store and returning the parameter value.
+    Note that all initialization arguments can optionally be provided or overwritten at runtime.
+    For authentication, there are two options: you can set the `AWS_CREDENTIALS` Prefect Secret
+    containing your AWS access keys, which will be passed directly to the `boto3` client, or you
+    can [configure your flow's runtime
+    environment](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#guide-configuration)
+    for `boto3`.
+    Args:
+        - parameter_name (str, optional): The name of the parameter to retrieve via SSM.
+        - boto_kwargs (dict, optional): Additional keyword arguments to forward to the boto client.
+        - **kwargs (dict, optional): Additional keyword arguments to pass to the
+            Task constructor.
+    """
+
+    def __init__(self, parameter_name: str = None, boto_kwargs: dict = None, **kwargs):
+        self.parameter_name = parameter_name
+
+        if boto_kwargs is None:
+            self.boto_kwargs = {}
+        else:
+            self.boto_kwargs = boto_kwargs
+
+        super().__init__(**kwargs)
+
+    @defaults_from_attrs("parameter_name")
```

Comment:
I think that we should add `boto_kwargs` here as well so that they can also be given at runtime.
```suggestion
    @defaults_from_attrs("parameter_name", "boto_kwargs")
```